tests/autoscaling/test_autoscaling_cluster_lib.py

```diff
@@ -1460,6 +1460,21 @@ def test_get_pool_slaves(self):
         ret = self.autoscaler.get_pool_slaves(mock_mesos_state)
         assert ret == {'id1': mock_mesos_state['slaves'][0]}
 
+    def test_instace_descriptions_for_ips_splits_ips(self):
+        with mock.patch(
+            'paasta_tools.autoscaling.autoscaling_cluster_lib.ClusterAutoscaler.describe_instances',
+            autospec=True,
+        ) as mock_describe_instances:
+            ips = list(range(567))
+
+            def mock_describe_instance(self, instance_ids, region, instance_filters):
+                return instance_filters[0]['Values']
+            mock_describe_instances.side_effect = mock_describe_instance
+
+            ret = self.autoscaler.instance_descriptions_for_ips(ips)
+            assert len(ret) == 567
```

Comment:
might be worth asserting `mock_describe_instances.call_count` too? but that's a nit-pick really.